addons/website_sale/static/tests/tours/website_sale_buy.js

```diff
@@ -53,14 +53,25 @@ tour.register('shop_buy_product', {
             trigger: '#cart_products tr:contains("Storage Box") a.js_add_cart_json:first',
         },
         {
-            content: "set one",
+            content: "set three", 
+            // cart_update_json take some time to answer, to prevent concurrent update, we set a new value of quantity.
+            // Like that the check amount work well
```

Comment:
This shouldn't be needed, 3 or 1 will do the same? the qty was 2 before this step